app/views/content/blog/my-career-change-to-teaching.md

```diff
@@ -0,0 +1,37 @@
+---
+title: My career change to teaching
+author: Ian Wallace
+date: "2022-02-03"
+images:
+  ian:
+    path: "media/images/content/blog/ian-wallace.jpg"
+    thumbnail_path: "media/images/content/blog/thumbnails/ian-wallace.jpg"
+description: |-
+  Ian Wallace had always wanted to be a teacher but didn’t feel ready when he finished university. After several different jobs and careers he finally took the plunge and hasn’t looked back.
+    
+keywords:
+  - teacher training
+  - trainee teacher course
+  - SCITT
+  - career change
+
+tags:
+  - becoming a teacher
+  - career change to teaching
+---
+
+$ian$
+
+When I was at school I always wanted to be a teacher, after the obvious football career, but when I finished university I didn’t think it was right for me. I had been in education a long time and worried I wasn’t mature enough. I went away and had several different jobs and careers at various levels of responsibility in retail, food and sports, including being self-employed a couple of times. 
```

Comment:
```suggestion
I always wanted to be a teacher when I was at school, after the obvious football career. But when I finished university I didn’t think it was right for me. I had been in education a long time and worried I wasn’t mature enough. I went away and had several different jobs and careers at various levels of responsibility in retail, food and sports, including being self-employed a couple of times. 
```
Is this first sentence a little complex? Could we break it up a little for readability?